CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Fixed
 * SRTP or condition disappear when non-generic type is used. [#2168](https://github.com/fsprojects/fantomas/issues/2168)
 * Multiline AbstractSlot without constraints introduces newline [#2175](https://github.com/fsprojects/fantomas/issues/2175)
+* Multiline parentheses objectExpression in indexer expression, 2176 [#2176](https://github.com/fsprojects/fantomas/issues/2176)
```

Comment:
4.7.6 was already release, you cannot add any items here.
Please use an `## [Unreleased]` section, see https://keepachangelog.com/en/1.0.0/

Reply:
Done, sorry.